<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotAnnotationWidget</class>
 <widget class="QWidget" name="PlotAnnotationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotAnnotationWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxPlotAnntotations">
     <property name="title">
      <string>Plot annotations:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTreeView" name="treeViewPlotAnnotations"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="thumbnail_preview_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Thumbnail preview:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelThumbnailPreview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Manual annotation:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="comboBoxAnnotationSubject"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxSAnnotationPredicate"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxAnnotationObject"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPlots">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Plots:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout2">
      <item row="0" column="5">
       <widget class="QListView" name="listViewPlots">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonAddPlot">
          <property name="text">
           <string>Add&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonAddAllPlot">
          <property name="text">
           <string>Add All&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonRemovePlot">
          <property name="text">
           <string>&lt;&lt;Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="6">
       <layout class="QVBoxLayout" name="verticalLayoutPlotEditor">
               <item row="0" column="0">
       <widget class="QWidget" name="PlotEditorsContainer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayoutPlotEditors">
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Plot type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxPlotType">
           <item>
            <property name="text">
             <string>timeseries</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>heatmap</string>
            </property>
           </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>x column:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditXColumn"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>y columns:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditYColumns"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Has header:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="checkBoxHasHeader">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Delimiter:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="comboBoxDelimiter">
           <item>
            <property name="text">
             <string>tab</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>comma</string>
            </property>
           </item>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButtonAnnotateCurrentPlot">
          <property name="text">
           <string>Annotate This Plot File</string>
          </property>
         </widget>
        </item>
       </layout>
       </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonAnnotateAllPlots">
          <property name="text">
           <string>Annotate All Plot Files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QTreeView" name="treeViewFileBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
